packages/ui5-proxy-middleware/src/base/types.ts

```diff
@@ -0,0 +1,34 @@
+import { NextFunction } from 'express';
+import { IncomingMessage } from 'http';
```

Comment:
Should we consider using `import type` here? I am still not fully sure where and when using `import types` actually has an advantage but I believe we started using `import type` in the Fiori tools coding whenever we just import types.
@Klaus-Keller might have an opinion here

Reply:
Yes, if it is just a type import we should use `import type`.

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/249/commits/8bac865e44992b25d7b97eb73b130c1ac661d6dc